src/components/Collections/Collection.tsx

```diff
@@ -258,7 +259,12 @@ const ImageCollection = ({
               }}
             />
           )}
-          <ReactionSettingsProvider settings={{ hideReactionCount: isContestCollection }}>
+          <ReactionSettingsProvider
+            settings={{
+              hideReactionCount: isContestCollection,
+              hideReactions: contestCollectionReactionsHidden(collection),
+            }}
```

Comment:
Just a nit, but this could be handled in the same method, we could call it `getContestCollectionReactionsSettings()`. But don't bother for now 😛